app/views/devise/registrations/_personal_details.html.erb

```diff
@@ -1,17 +1,17 @@
 <%= form_for(resource, namespace: current_user.id, as: resource_name, url: registration_path(resource_name), html: {method: :put, id: 'personal_details_registration_form' }) do |f| %>
-  <p class="form-control-static">
-    <%= sanitize _("Please note that your email address is used as your username. If you change this, remember to use your new email address on sign in.") %>
-  </p>
-
-  <p class="form-control-static"><%= _('You can edit any of the details below.') %></p>
-  <%= hidden_field_tag :unlink_flag, "false", id: 'unlink_flag' %>
 
   <div class="form-group col-xs-8">
     <%= f.label(:email, _('Email'), class: 'control-label') %>
-    <%= f.email_field(:email, class: "form-control", "aria-required": true, value: @user.email) %>
+    <%= f.email_field(:email, class: "form-control", "aria-required": true, value: @user.email, "disabled": true) %>
     <%= hidden_field_tag :original_email, @user.email %>
   </div>
 
+  <div class="form-group col-xs-12">
+    <p class="form-control-static"><%= _('You can edit any of the details below.') %></p>
+  </div>
```

Comment:
I almost missed this part! It will make more sense to the users that they can change the rest of the details.